apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/WebviewErrorDrawer/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Trans } from "react-i18next";
+import styled from "styled-components";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import Box from "~/renderer/components/Box";
+import { useGetSwapTrackingProperties } from "../../utils/index";
+import { Text } from "@ledgerhq/react-ui";
+import ErrorNoBorder from "~/renderer/icons/ErrorNoBorder";
+
+const ContentBox = styled(Box)`
+  display: flex;
+  justify-content: center;
+  height: 100%;
+`;
```

Comment:
styling, add an empty line between the two components

Reply:
Ledger live do not have that structure and it is really small file, I am not sure if it is helping or making it adds complexity
<img width="465" alt="Screenshot 2023-10-26 at 09 48 57" src="https://github.com/LedgerHQ/ledger-live/assets/38540290/eaf21750-36eb-4ff7-980d-468a72fe9248">